packages/@uppy/companion/src/server/provider/zoom/index.js

```diff
@@ -209,9 +211,15 @@ class Zoom extends Provider {
         modifiedDate: adapter.getDateFolderModified(end),
         size: null
       })
+
+      if (end.isSame(limit, 'month')) {
```

Comment:
is this always the condition at which the loop will stop? If so, should we simply change the while loop to ` while (end.isAfter(limit, 'month')) {` or something similar instead?

Reply:
This was a bit confusing, but this had to be where it was in order to push in the earliest month correctly.  I've made changes to update the initial start/end months before entering the loop, and fixing the boundary condition so it's hopefully a bit clearer.